### PR TITLE
Feature/tineye

### PR DESCRIPTION
**Pull request by ExitoLab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,6 +363,7 @@ These can be useful for osint and social engineering.
 - [Yandex Image](https://www.yandex.com/images/) - Yandex Image Search
 - [Facecheck.id](https://facecheck.id/) - Search for people by photo and verify you are talking to the person they claim to be
 - [Bing Visual Search](https://www.bing.com/visualsearch) - See it, search it
+- [Tineye Search](https://tineye.com/) - Reverse Image Search
```
Comment (edoardottt) [inline README.md:366]: This should be:
```
- [Tineye](https://tineye.com/) - Image search and recognition company
```

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ExitoLab) [reply README.md]: @edoardottt thanks. I have worked on this

Comment (edoardottt) [reply README.md]: here you're trying to add another search engine, please correct that + conflicting readme

Comment (ExitoLab) [reply README.md]: I have fixed this @edoardottt

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**